### PR TITLE
statistics: replace loads with []float64 in peer info 

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve?

replace loads with []float64 in peer info 

### Check List

Tests

- Unit test

### Release note

```release-note
None
```

### CODE REVIEW

**Review by Yisaer — APPROVED** [review]

File: server/statistics/util.go [hunk 1]
```diff
@@ -15,6 +15,9 @@ package statistics
 
 import (
 	"fmt"
+
+	"github.com/tikv/pd/server/core"
+
```
Comment (rleungx) [inline server/statistics/util.go:20]: remove the empty line

File: server/statistics/util.go [hunk 2]
```diff
@@ -27,3 +30,13 @@ const (
 func storeTag(id uint64) string {
 	return fmt.Sprintf("store-%d", id)
 }
+
+func GetLoads(r *core.RegionInfo) []float64 {
```
Comment (rleungx) [inline server/statistics/util.go:34]: Is this function belong to region?

Comment (lhy1024) [reply server/statistics/util.go:34]: yes, It is placed here to avoid circular references between core and statistic

Comment (rleungx) [reply server/statistics/util.go:34]: I think it can be removed now?

Comment (lhy1024) [reply server/statistics/util.go:34]: done

File: server/cluster/cluster.go [hunk 3]
```diff
@@ -589,8 +596,7 @@ func (c *RaftCluster) processRegionHeartbeat(region *core.RegionInfo) error {
 	interval := reportInterval.GetEndTimestamp() - reportInterval.GetStartTimestamp()
 	for _, peer := range region.GetPeers() {
 		peerInfo := core.NewPeerInfo(peer,
-			region.GetBytesWritten(), region.GetKeysWritten(),
-			0, 0,
+			region.GetLoads(),
```
Comment (rleungx) [inline server/cluster/cluster.go:599]: Can we directly get loads here?

**Review by rleungx — APPROVED** [review]

File: server/core/region.go [hunk 4]
```diff
@@ -960,6 +960,26 @@ func (r *RegionsInfo) GetFollower(storeID uint64, region *RegionInfo) *RegionInf
 	return nil
 }
 
+// GetLoads returns loads from region
+func (r *RegionInfo) GetLoads() []float64 {
+	return []float64{
+		float64(r.GetBytesRead()),
```
Comment (nolouch) [inline server/core/region.go:966]: Can this unity to `GetLoadbyKind`?

**Review by nolouch — APPROVED** [review]
LGTM

File: server/core/region.go (hunk 4 above)
Comment (lhy1024) [reply server/core/region.go:966]: I will move `kind.go` of statistic to core in a later pr to avoid circular dependencies

**Pull request merged**